src/dapp/CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Dapp debug respects DAPP_LINK_TEST_LIBRARIES
+- Dapp debug respects `DAPP_LINK_TEST_LIBRARIES`
+- Dapp remappings ignores non-directories in `DAPP_LIB`
```

Comment:
Can you move this to an `## Unreleased` heading pls

Reply:
oop